tests/chainer_tests/functions_tests/pooling_tests/test_upsampling_2d.py

```diff
@@ -55,12 +55,12 @@ def check_forward(self, y):
                 testing.assert_allclose(up_y, 0)
 
     def test_forward_cpu(self):
-        self.pooled_y.to_cpu()
+        self.pooled_y.to_cpu(allow_unchain=True)
```

Comment:
I think this `allow_unchain=True` is undesirable, because it changes the state of `pooled_y` and it's nonessential for the test. How about calling `self.pooled_y.unchain()` in `setUp` (or TODO)?

Reply:
Okay, fixed to call `unchain` in `setUp`.